cmd/plasmacli/subcmd/query/info.go

```diff
@@ -40,28 +40,23 @@ var infoCmd = &cobra.Command{
 			return err
 		}
 
-		var utxos []store.UTXO
+		var utxos []store.TxOutput
 		if err := json.Unmarshal(data, &utxos); err != nil {
 			return fmt.Errorf("unmarshaling json query response: %s", err)
 		}
 
 		for i, utxo := range utxos {
 			fmt.Printf("UTXO %d\n", i)
-			fmt.Printf("Position: %s, Amount: %s, Spent: %t\n", utxo.Position, utxo.Output.Amount.String(), utxo.Spent)
-
+			fmt.Printf("Position: %s, Amount: %s, Spent: %t\nSpender Hash: %s\n", utxo.Position, utxo.Output.Amount.String(), utxo.Spent, utxo.SpenderTx)
+			fmt.Printf("Transaction Hash: 0x%x\nConfirmationHash: 0x%x\n", utxo.TxHash, utxo.ConfirmationHash)
+			/*
+			// TODO: Add --verbose flag that if set will query TxInput and print InputAddresses and InputPositions as well
```

Comment:
Verbose flag should be in another PR that is a persistent global flag amongst all commands